search/query_params.hpp

```diff
@@ -110,7 +113,13 @@ class QueryParams
 
   inline Langs & GetLangs() { return m_langs; }
   inline Langs const & GetLangs() const { return m_langs; }
-  inline bool LangExists(int8_t lang) const { return m_langs.count(lang) != 0; }
+
+  inline bool LangExists(int8_t lang) const
```

Comment:
Probably we need to create a wrapper around SmallSet, with additional checks for unsupported languages. @mpimenov WDYT?